File: questApplication/src/main/java/ru/nekit/android/qls/utils/Vibrate.java
package ru.nekit.android.qls.utils;

import android.content.Context;
import android.os.Vibrator;
import android.support.annotation.NonNull;

public class Vibrate {

    public static void make(@NonNull Context context, long time) {
        ((Vibrator) context.getSystemService(Context.VIBRATOR_SERVICE)).vibrate(time);
    }

}

File: questPresentation/src/main/java/ru/nekit/android/qls/utils/Vibrate.kt
package ru.nekit.android.qls.utils

import android.content.Context
import android.os.Vibrator

object Vibrate {

    fun make(context: Context, time: Long) {
        (context.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator).vibrate(time)
    }

}
